java/fury-core/src/main/resources/META-INF/NOTICE

```diff
@@ -6,59 +6,33 @@ The Apache Software Foundation (http://www.apache.org/).
 
 ================================================================
 
-This product includes a number of Dependencies with separate copyright notices
+This product includes a number of Dependencies/Code with separate copyright notices
 and license terms. Your use of these submodules is subject to the terms and
 conditions of the following licenses.
 
 ================================================================
 
-================================================================
-Apache-2.0 licenses
-================================================================
-The following components are provided under the Apache-2.0 License. See project link for details.
-The text of each license is the standard Apache 2.0 license.
-
-* guava (https://github.com/google/guava)
-    Files:
-      java/fury-core/src/main/java/org/apache/fury/util/Preconditions.java
-
 * spark (https://github.com/apache/spark)
```

Comment:
I don't think we need to name the files in the NOTICE - we have the file names in the LICENSE file.

The NOTICE is generally meant to be short. It is mainly for copyright details.

Reply:
Thanks, updated